android/app/src/main/res/values-hu/strings.xml

```diff
@@ -714,6 +714,24 @@
 	<string name="osm_wiki_about_url">https://wiki.openstreetmap.org/wiki/Hu:Névjegy</string>
 	<!-- Tip #00 -->
 	<string name="app_tip_00">Köszönjük, hogy használja a közösség által készített térképeinket!</string>
+	<!-- App tip #01 -->
+	<string name="app_tip_01">Adományaival és támogatásával létrehozhatjuk a világ legjobb térképes alkalmazását!</string>
+	<!-- App tip #02 -->
+	<string name="app_tip_02">Tetszik az alkalmazásunk? Kérjük, adományozzon a fejlesztés támogatására! Még nem tetszik? Kérjük, jelezze felénk, és mi megoldjuk!</string>
```

Comment:
Use "támogatása érdekében" instead of "támogatására"

Reply:
Final wording: `Tetszik az alkalmazásunk? Kérjük, adományozzon a fejlesztés támogatása érdekében! Még nem tetszik? Kérjük, jelezze felénk, és mi megoldjuk!`